ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -104,6 +104,10 @@ public FreeControlForm(ControlRoom new_controlRoom, int new_rtId, RTControlFormD
 
             DatabaseOperations.AddAppointment(CurrentAppointment);
 
+            // Initialize STOP Telescope (stopRT) button
+            stopRT.BackColor = System.Drawing.Color.Red;
+            stopRT.Enabled = true;
```

Comment:
I just realized that we don't need either of these now that the button color isn't changing and isn't being disabled. Instead, you can set these fields to be the default in the control form GUI design properties.

- [ ] Move initialization settings to default GUI properties

On that note, I also noticed that the `FlatStyle` isn't the same as the other buttons, so the buttons look different when hovered over. Change the `FlatStyle` property to `Flat` to stick with the same design.

- [ ] Change `FlatStyle` to `Flat`